### PR TITLE
Added tfvars and backend files for DNS and Frontdoor

### PR DESCRIPTION
**Pull request by enformatique11**

### CODE REVIEW

**Comment by enformatique11** [comment]
[Custom Domains](https://trello.com/c/6XlNPiiH/426-custom-domains)

File: custom_domains/infrastructure/workspace_variables/trngen.tfvars.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        }
+    },
+    "tags": {
+        "Parent Business": "Teaching Regulation Agency",
+        "Portfolio": "Early Years and Schools Group",
+        "Product": "Database of Qualified Teachers",
+        "Service": "Teacher Services",
+        "Service Line": "Teaching Workforce",
+        "Service Offering": "Database of Qualified Teachers",
+        "Environment": "Prod"
```
Comment (NickGraham101) [inline custom_domains/infrastructure/workspace_variables/trngen.tfvars.json]: lowercase p for `prod`

Comment (enformatique11) [reply custom_domains/infrastructure/workspace_variables/trngen.tfvars.json]: fixed

**Review by NickGraham101 — APPROVED** [review]

File: custom_domains/infrastructure/workspace_variables/trngen.tfvars.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+    "hosted_zone": {
+        "trn-generation-api.education.gov.uk/": {
```
Comment (NickGraham101) [inline custom_domains/infrastructure/workspace_variables/trngen.tfvars.json]: Remove backslash

Comment (enformatique11) [reply custom_domains/infrastructure/workspace_variables/trngen.tfvars.json]: fixed

**Comment by enformatique11** [comment]
[Custom Domains](https://trello.com/c/6XlNPiiH/426-custom-domains)

**Pull request merged**